src/main/java/com/ericsson/ei/subscriptionhandler/SubscriptionRepeatDbHandler.java

```diff
@@ -0,0 +1,211 @@
+/*
+   Copyright 2018 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.ei.subscriptionhandler;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+
+import com.ericsson.ei.mongodbhandler.MongoDBHandler;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.mongodb.BasicDBObject;
+import com.mongodb.util.JSON;
+
+import lombok.Getter;
+import lombok.Setter;
+
+
+@Component
+public class SubscriptionRepeatDbHandler {
+
+	static Logger log = (Logger) LoggerFactory.getLogger(SubscriptionRepeatDbHandler.class);
+
+	
+    @Autowired
+    public MongoDBHandler mongoDbHandler;
```

Comment:
make it private

Reply:
Can't set this to private,, that will fail unit tests since we configure this MongoDbHandler with a Embedded MongoDb from the unit tests.